cmd/konvoy-image-wrapper/cmd/create-package-bundle.go

```diff
@@ -142,23 +141,30 @@ func (r *Runner) CreatePackageBundle(args []string) error {
 			return err
 		}
 	}
+
+	fetchKubernetesRPMs := true
 	kubernetesVersion := kubernetesVersionFlag
 	if kubernetesVersion == "" {
 		kubernetesVersion, err = getKubernetesVerisonFromAnsible()
 		if err != nil {
 			return err
 		}
+		// if we are getting the default version from ansible, we don't need to modify this.
+		fetchKubernetesRPMs = false
 	}
 	if eusReposFlag {
 		r.env["EUS_REPOS"] = "true"
 	}
+	if fetchKubernetesRPMs {
+		r.env["KUBERNETES_REPOS"] = "true"
```

Comment:
nit: maybe `ENABLE_KUBERNETES_REPOS`?

Reply:
lets leave it. its still pretty clear and we don't want to run all these tests again over one word.